Sources/XcodeGenKit/ProjectGenerator.swift

```diff
@@ -96,6 +96,72 @@ public class ProjectGenerator {
                         archiveAction: archiveAction)
     }
 
+    func generateBreakpointList(_ breakpoints: [Breakpoint]) throws -> XCBreakpointList {
+        return XCBreakpointList(type: "4", version: "2.0", breakpoints: try breakpoints.flatMap({ try generateBreakpointProxy($0) }))
+    }
+
+    private func generateBreakpointProxy(_ breakpoint: Breakpoint) throws -> XCBreakpointList.BreakpointProxy {
+        var extensionID: XCBreakpointList.BreakpointProxy.BreakpointExtensionID
+        switch breakpoint.extensionID {
+            case "file": extensionID = .file
+            case "exception": extensionID = .exception
+            case "swiftError": extensionID = .swiftError
+            case "openGLError": extensionID = .openGLError
+            case "symbolic": extensionID = .symbolic
+            case "ideConstraintError": extensionID = .ideConstraintError
+            case "ideTestFailure": extensionID = .ideTestFailure
+            default: throw SpecValidationError.ValidationError.invalidBreakpointExtensionID(breakpoint.extensionID)
+        }
+
+        let xcbreakpoint = XCBreakpointList.BreakpointProxy.BreakpointContent(enabled: breakpoint.enabled,
```

Comment:
Unless we give the breakpoint spec some smarts, it looks like it's just shadowing the struct in xcproj. In that case we can use those same types. What do you think?